x/evm/types/tx.go

```diff
@@ -1,10 +1,32 @@
 package types
 
 import (
+	math "math"
```

Comment:
It's hard to get the purpose of such naming, would be great to add comment with explanation

Reply:
```suggestion
	"math"
```